tools/envoy_headersplit/README.md

```diff
@@ -0,0 +1,12 @@
+# Envoy Header Split
+Tool for spliting monolithic header files in Envoy to speed up compilation
```

Comment:
How often do you think Envoy developers should be running this? Is this a once a year kind of cleanup?

Reply:
I think the period could be several months or a year. It should be long enough until we get some monolithic mock headers appeared in Envoy repository.